va/va.h

```diff
@@ -3922,6 +3922,38 @@ VAStatus vaQuerySurfaceError(
  */
 #define VA_FOURCC_IYUV          0x56555949
 
+/** Y8: 8-bit greyscale.
+ */
+#define VA_FOURCC_Y8            0x20203859
+
+/** Y16: 16-bit greyscale.
+ */
+#define VA_FOURCC_Y16           0x20363159
+
+/** VYUV: packed 8-bit YUV 4:2:2.
+ *
+ * Four bytes per pair of pixels: V, Y, U, V.
+ */
+#define VA_FOURCC_VYUY          0x59555956
+
+/** YVYU: packed 8-bit YUV 4:2:2.
+ *
+ * Four bytes per pair of pixels: Y, V, Y, U.
+ */
+#define VA_FOURCC_YVYU          0x55595659
+
+/** ARGB: packed 16-bit BGRA.
+ *
+ * Four bytes per pixel: alpha, red, green, blue.
+ */
+#define VA_FOURCC_ARGB64        0x34475241
+
+/** ABGR: packed 16-bit RGBA.
+ *
+ * Four bytes per pixel: alpha, blue, green, red.
+ */
+#define VA_FOURCC_ABGR64        0x34474241
```

Comment:
Same as the above.

Reply:
change the command following the reviewed